website/src/lib/oasst_api_client.ts

```diff
@@ -16,7 +16,13 @@ export class OasstError {
 }
 
 export class OasstApiClient {
-  constructor(private readonly oasstApiUrl: string, private readonly oasstApiKey: string) {}
+  oasstApiUrl: string;
+  oasstApiKey: string;
+
+  constructor(oasstApiUrl: string, oasstApiKey: string) {
+    this.oasstApiUrl = oasstApiUrl;
+    this.oasstApiKey = oasstApiKey;
+  }
```

Comment:
Why do we need to change this? Both codes are the same right?

Reply:
For reasons I haven't quite figured out, storybook breaks with `private readonly`.  With the previous code storybook stories can't present.

Reply:
maybe because some of our storybook stories are `.jsx` instead of `.tsx`, and therefore typescript features are not supported?